README.md

```diff
@@ -87,3 +72,25 @@ server {
 }
 ```
 
+**Development**
```

Comment:
This can be reverted once we drop the `make debug` and `docker-compose.yml` file.

In general, we want to keep inline with upstream (so changing how we develop an upstream project isn't something we recommend).